lib/protocol/encoder.js

```diff
@@ -0,0 +1,20 @@
+var msgpack = require('msgpack2');
```

Comment:
I think msgpack5 is now the latest message-pack library. Not a requirement though.